include/industry_standard/spdm.h

```diff
@@ -21,6 +21,7 @@
 
 #define SPDM_NONCE_SIZE 32
 #define SPDM_RANDOM_DATA_SIZE 32
+#define SPDM_REQ_CONTEXT_SIZE 8
```

Comment:
This is specific to `CHALLENGE` / `CHALLENGE_AUTH`.

Reply:
NO. MEASUREMENT also need this.

Reply:
Ok.